runtests.py

```diff
@@ -111,7 +111,7 @@ def add_pytest(self, name: str, pytest_args: List[str], coverage: bool = False)
         else:
             args = [sys.executable, '-m', 'pytest'] + pytest_args
 
-        self.waiter.add(LazySubprocess(full_name, args, env=self.env, passthrough=True),
+        self.waiter.add(LazySubprocess(full_name, args, env=self.env, passthrough=self.verbosity),
```

Comment:
I can handle `-j` and `-q`/`-v` this way, but we'll need to be careful with the logic (e.g., if there's `-v` and `-p -v`, should those `-v` be added up, or does one supercede the other?). 

The other option is not to pass through `-j` / `-q` / `-v`, and just let the user set pytest options directly using the existing `-p` flag which passes the next argument directly to pytest: `python runtests.py -j8 -p -n4` currently means "run pytest with 4 parallel workers, and the other tests with 8".